apps/backpack/views.py

```diff
@@ -45,24 +45,52 @@ def __init__(self, x, y, width, height, radius, text, strokecolor, fillcolor, st
         self.studyload = studyload
         self.esco = esco
 
+    def split_text(self, text, max_width):
+        words = text.split()
+        lines = []
+        current_line = ""
+
+        for word in words:
+            test_line = f"{current_line} {word}".strip()
+            if self.canv.stringWidth(test_line, 'Helvetica-Bold', 10) <= max_width:
+                current_line = test_line
+            else:
+                lines.append(current_line)
```

Comment:
If there is only one word, but this word is already too long, this would add an empty line and then the word. I think it would make sense to add an `if` herer, only appending the `current_line` if it's non-empty